Geometry/MuonNumbering/src/RPCNumberingScheme.cc

```diff
@@ -34,6 +42,13 @@ void RPCNumberingScheme::initMe(const MuonDDDConstants& muonConstants) {
 }
 
 int RPCNumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) {
+  const int mylevel = num.getLevels();
+  //cout<<"------------myDEBUG DDD-------------------------------------------"<<endl;
```

Comment:
@slomeo please avoid commented printout and use the MessageLogger instead (possibly with pre-processor conditions)

Reply:
@fabiocos I deleted all not necessary "cout" and "include"

> @slomeo please avoid commented printout and use the MessageLogger instead (possibly with pre-processor conditions)